tests/errors.rs

```diff
@@ -20,10 +21,12 @@ fn test_error_parse() {
     let mut c = Config::default();
     let res = c.merge(File::new("tests/Settings-invalid", FileFormat::Toml));
 
+    let path : PathBuf = ["tests", "Settings-invalid.toml"].iter().collect();
+
     assert!(res.is_err());
     assert_eq!(
         res.unwrap_err().to_string(),
-        "failed to parse datetime for key `error` at line 2 column 9 in tests/Settings-invalid.toml".to_string()
+        format!("failed to parse datetime for key `error` at line 2 column 9 in {}", path.to_str().unwrap())
```

Comment:
nit: paths have a `.display()` to avoid unwraps like this

Reply:
Thank you for the hint!